lib/Common/Memory/Recycler.h

```diff
@@ -2372,7 +2375,8 @@ struct ForceLeafAllocator<RecyclerNonLeafAllocator>
     typedef RecyclerLeafAllocator AllocatorType;
 };
 
-#ifdef PROFILE_EXEC
+// TODO: enable -profile for GC phases
+#if defined(PROFILE_EXEC) && 0
```

Comment:
What is blocking from making this work? 
AFAIK, this used to work, but something might have broke it?